src/components/grid-aware/ImageHeader/ImageHeader.module.css

```
@@ -90,6 +90,23 @@
   margin-top: 20px;
 }
 
+.descriptionWrapper :first-child {
+  margin: 0;
+}
+
+.descriptionWrapper :not(:first-child):not(li) {
```

Instead of using the `:not(li)` selector, I think what you really want is to use a direct child selector, `>`:

```suggestion
.descriptionWrapper > :not(:first-child) {
```

By default, nested selectors will select items that are arbitrarily deep, but here, I assume you want to select the first direct child of `.descriptionWrapper`, but not the first child of any level of the tree.